x-pack/test/functional/apps/lens/group3/open_in_lens/tsvb/timeseries.ts

```diff
@@ -83,5 +87,120 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       expect(await queryBar.getQueryString()).to.equal('machine.os : ios');
     });
+
+    it('should draw a reference line', async () => {
+      await visualBuilder.createNewAggSeries();
+      await visualBuilder.selectAggType('Static Value');
+      await visualBuilder.setStaticValue(10);
+
+      await header.waitUntilLoadingHasFinished();
+
+      const button = await testSubjects.find('visualizeEditInLensButton');
+      await button.click();
+      await lens.waitForVisualization('xyVisChart');
+      await retry.try(async () => {
+        const layers = await find.allByCssSelector(`[data-test-subj^="lns-layerPanel-"]`);
+
+        const referenceLineDimensions = await testSubjects.findAllDescendant(
+          'lns-dimensionTrigger',
+          layers[0]
+        );
+        expect(referenceLineDimensions).to.have.length(1);
+        expect(await referenceLineDimensions[0].getVisibleText()).to.be('Static value: 10');
+
+        const dimensions = await testSubjects.findAllDescendant('lns-dimensionTrigger', layers[1]);
+        expect(dimensions).to.have.length(2);
+        expect(await dimensions[0].getVisibleText()).to.be('@timestamp');
+        expect(await dimensions[1].getVisibleText()).to.be('Count of records');
+      });
+    });
+
+    it('should convert metric with params', async () => {
+      await visualBuilder.selectAggType('Counter Rate');
+      await visualBuilder.setFieldForAggregation('machine.ram');
+
+      await header.waitUntilLoadingHasFinished();
+
+      const button = await testSubjects.find('visualizeEditInLensButton');
+      await button.click();
+      await lens.waitForVisualization('xyVisChart');
+      await retry.try(async () => {
+        const layers = await find.allByCssSelector(`[data-test-subj^="lns-layerPanel-"]`);
```

Comment:
there's a method on the page object for this: https://github.com/elastic/kibana/blob/91dbdc7888a148c3bde671c97624f46ff838026b/x-pack/test/functional/page_objects/lens_page.ts#L857

Reply:
@flash1293, thanks. I've fixed the code according to your suggestion.